forge.config.js

```diff
@@ -51,6 +51,60 @@ async function downloadPandoc (platform, arch) {
   })
 }
 
+/**
+ * This function returns the full path and filename to the library specified
+ * by `libraryName`. Uses ldconfig to determine the library location because
+ * libraries location varies.
+ *
+ * @param   {string}  libraryName  The name of the library.
+ * @returns {string}  The full path of the matching library. If library is not
+ *                    found, throws an Error.
+ */
+async function getLibraryPath (libraryName) {
+  return new Promise((resolve, reject) => {
+    const shellProcess = spawn('/sbin/ldconfig', ['-p'])
+    let out = ''
+    shellProcess.stdout.on('data', (data) => {
+      out += data.toString()
+    })
+    shellProcess.on('close', (code, signal) => {
+      if (code !== 0) {
+        reject(new Error(`Failed to run ldconfig: Process quit with code ${code}`))
+      } else {
```

Comment:
I don't think you need the else here, since a reject automatically aborts function execution, or? (Just asking because down in L84 you do it)